### DISCUSSION TITLE
Support import of Evernote .enex files

### DISCUSSION THREAD

**Discussion by SteveGBuck** [jonaswinkler/paperless-ng#279] [Feature Requests]
Loving this project!  Just got my docker containers running and doing some testing and am really liking the approach and workflow.

One challenge I have however is that I've been paperless since 2016 and have over 6500 documents in Evernote already that I would want to figure out how to transfer over to paperless-ng.

Would there be any interest in supporting import of enex files?

I've yet to explore it further, but don't think there is an easy way to extract PDFs and Jpgs from my notes without doing it on a 1 on 1 basis.

**Comment by SteveGBuck** [comment]
For reference for anyone else that stumbles on this post. I found a way to do this. 

Installing the [legacy](https://help.evernote.com/hc/en-us/articles/360052560314-Install-an-older-version-of-Evernote) version of Evernote restores the ability to bulk save your attached files within Evernote.

I've extracted my documents in batches based on my Evernote tags / notebooks and saved them into nested folders. e.g. Banking > Credit Cards > Capital One.

I then enabled PAPERLESS_CONSUMER_RECURSIVE and PAPERLESS_CONSUMER_SUBDIRS_AS_TAGS which assigns my folder names as tags to my documents during import. So with the example above my Capital One documents now have the tags: Banking, Credit Cards, Capital One assigned.

From there I can now use tag filters to bulk assign any correspondents and document types.

**Reply by jonaswinkler** [reply]
Glad you got it figured out. I never thought that this feature could be used in this way.

**Reply by zombiehoffa** [reply]
Curious why you would want to store your evernote documents in something like paperless  vs something like trilium (https://github.com/zadam/trilium/)? Maybe I used evernote differently but it was definitely a living document sort of place for me (which is why I love trilium as a replacement) whereas this project is much more about  archival uses (bank statements, very important emails, manuscripts, etc) for me.

**Reply by SteveGBuck** [reply]
I've never seen trillium before, but looks interesting!

Actually I solely use Evernote as a paperless document archive - it's not my go to note taking platform. It actually works pretty well for this purpose but am bound to the increasing subscription costs and have always been nervous about how secure my data is.
